### DISCUSSION TITLE
transform into SRT and  with max_line_count and max_line_width

### DISCUSSION THREAD

**Discussion by jravolio** [openai/whisper#1586] [Q&A] [answered]
i'm pretty new to using whisper, sorry if my question is too noob. How could i export as SRT and specify the max_line_count and max_line_width in a python code? I tried to search for those functions on the util.py but got confused. Right now i only managed to achieve this by using the cli. Here is my current python code:

```
audio_file= open("audio.mp3", "rb")
model = whisper.load_model("small")
result = model.transcribe("audio.mp3",initial_prompt="prompt", word_timestamps=True)
print(result["text"])
```

Thanks in advance!

**Comment by charl-ie** [comment]
I would love to hear the answer to this! I am having trouble too

**Reply by ryanheise** [reply]
https://github.com/openai/whisper/discussions/98#discussioncomment-6459910

**Comment by maciusr** [comment, marked as answer]
This is how I would do it expanding your code:
```
audio_file= open("audio.mp3", "rb")
model = whisper.load_model("small")
result = model.transcribe("audio.mp3",initial_prompt="prompt", word_timestamps=True)
print(result["text"])
word_options = {
        "highlight_words": False,
        "max_line_count": 50,
        "max_line_width": 3
    }
srt_writer = get_writer("srt", folder)
    srt_writer(result, 'audio.mp3', word_options)
```
Added word_options, that allow you to define all those additional options, then use get_writer to convert result to SRT. Hope that helps!

BTW You will need additional import 
```
from whisper.utils import get_writer
```

**Reply by jravolio** [reply]
Thanks a lot! it works very well for me!
